packages/fxa-shared/package.json

```diff
@@ -112,7 +112,8 @@
     "reflect-metadata": "^0.1.13",
     "rxjs": "^7.2.0",
     "stripe": "^8.191.0",
-    "superagent": "^6.1.0"
+    "superagent": "^6.1.0",
+    "typesafe-joi": "^2.1.0"
```

Comment:
[`typesafe-joi`](https://www.npmjs.com/package/typesafe-joi) has a warning label on it saying it’s a WIP without unit tests -- do we want to use this package in a production application? What are the risks?

I do see we are already using it in fxa-event-broker and fxa-support-panel...

Reply:
Yea, its worked well enough with no issues so far, but its possible a more complex validation schema might not work with it.